ast_tools/passes/ssa.py

```diff
@@ -477,8 +504,16 @@ def visit_Assign(self, node: cst.Assign) -> tp.Optional[bool]:
     def leave_Assign(self,
             original_node: cst.Assign,
             updated_node: cst.Assign) -> cst.Assign:
+        if (len(updated_node.targets) == 1 and
+                match.matches(updated_node.targets[0],
+                              match.AssignTarget()) and
```

Comment:
Good catch, before I was matching on match.Name() inside the assign target, but removed that since I wanted to support Attributes as well, so I don't think this is a necessary condition anymore.